Source/OrchBase/Handlers/CmDepl.cpp

```diff
@@ -41,7 +41,6 @@ void CmDepl::Cm_App(Cli::Cl_t clause)
     /* Ensure that all graph instances are built before proceeding. */
     for (graphIt = graphs.begin(); graphIt != graphs.end(); graphIt++)
     {
-#warning "CmBuil::Cm_Deploy: No logic defined for checking whether an application has been built. Deploy-check does not interact with the Composer."
```

Comment:
How has this ex-warning been addressed? :sweet_potato:

Reply:
The GraphInstance has a "built" flag in it that is managed by Composer. CmDepl checks this on (new) line 44